packages/webapp/src/pages/Account/InviteSignup.tsx

```diff
@@ -89,6 +92,30 @@ export const InviteSignup: React.FC = () => {
         return null;
     }
 
+    if (isLogged && isLogged.email !== data.invitation.email) {
+        return (
+            <DefaultLayout>
+                <div className="flex flex-col justify-center">
+                    <div className="flex flex-col justify-center mt-4 gap-4">
+                        <Info size={20} color="orange" classNames="text-xs">
+                            This invitation was sent to a different email. Please use the correct account or logout
```

Comment:
Please logout and use the correct account ?

Reply:
indeed, better 👌🏻